packages/rest-api/src/constants/bridgeable.ts

```diff
@@ -1431,3 +1431,20 @@ export const WLD: BridgeableToken = {
   imgUrl:
     'https://synapse-interface-worldchain.sanguine-fe.pages.dev/_next/static/media/worldchain.62d1dfd2.svg',
 }
+
+export const BNB: BridgeableToken = {
```

Comment:
Tests in `rest-api` will be behind since there is a new token. So can increment the number in those tests and will pass.